<launch>
	<arg name="rviz" default="true" />

        <param name="voxel_size" type="double" value="0.3" />
	<param name="file_path" type="string" value="$(find balm2)" />
        <node pkg="balm2" type="benchmark_test" name="benchmark_test" output="screen" >
            <param name="bagfile" type="string" value="/home/south/data/calib0620_cam4/post/internal/south-slam-raw.bag"/>
            <param name="sensor_json" type="string" value="/home/south/data/calib0620_cam4/sensor.json"/>
            <param name="trajectoryfile" type="string" value="/home/south/data/calib0620_cam4/post/internal/trajectory.bag"/>
        </node>

	<group if="$(arg rviz)">
		<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find balm2)/rviz_cfg/back.rviz" />
	</group>  

</launch>
